src/pint/models/timing_model.py

```diff
@@ -304,6 +304,40 @@ def phase_funcs(self):
             pfs += p.phase_funcs_component
         return pfs
 
+    @property
+    def is_binary(self):
+        """Does the model describe a binary pulsar? (True or False)"""
+        return any(x.startswith("Binary") for x in self.components.keys())
+
+    def orbital_phase(self, barytimes, anom="mean"):
+        """Return orbital phase (in radians) at barycentric MJD times"""
```

Comment:
This docstring needs to be expanded. Should define the possible choices for `anom` and should use the standard docstring style defining Parameters and Returns.